<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="buttons_frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>300</width>
        <height>250</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton0">
       <property name="geometry">
        <rect>
         <x>75</x>
         <y>200</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonpoint">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton1">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>200</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="equals">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>200</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
      <widget class="QPushButton" name="percentage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton3">
       <property name="geometry">
        <rect>
         <x>75</x>
         <y>150</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton6">
       <property name="geometry">
        <rect>
         <x>75</x>
         <y>100</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton9">
       <property name="geometry">
        <rect>
         <x>75</x>
         <y>50</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exponential">
       <property name="geometry">
        <rect>
         <x>75</x>
         <y>0</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>x²</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton7">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
      <widget class="QPushButton" name="squareroot">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>√</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clear">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>0</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
      <widget class="QPushButton" name="add">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>150</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="subtract">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>100</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="multiply">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>50</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
      <widget class="QPushButton" name="backspace">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>0</y>
         <width>75</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px
</string>
       </property>
       <property name="text">
        <string>⌫</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="screen_frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>39</y>
        <width>300</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="display">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-1</y>
         <width>300</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 45px</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="converter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Currency Converter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30px</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Amout:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>From:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxfrom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>USD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JPY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GBP</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>From:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxto">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>USD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JPY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GBP</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="result">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 45px</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="convertbutton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
